### DISCUSSION TITLE
When an old game engine goes open source, should it be included here?

### DISCUSSION THREAD

**Discussion by kparal** [opengaming/osgameclones#1504] [Q&A] [answered]
Today, Little Big Adventure 1 and 2 went open source:
https://blog.2point21.com/p/devlog-releasing-lba-open-source

The developers even created separate repositories for community-based development. I wonder, should those repos be included in osgameclones? They don't fit into the Remake/Clone/Similar/Tool category. But it feels weird to not include them just because they are the original source code and not a community remake. Thoughts?

(Btw, LBA is already in osgameclones, because a remake for it exists).

**Comment by cxong** [comment, marked as answer]
Yes; we can use the `remake` type. It's not perfect but we already have a few repos that use the same type for the same purpose. For example: virtually all the quake and doom clones use code from the original games open sourced later. Sometimes they have minimal changes, sometimes they add lots of new features, so it is hard to draw a line. Easier to just include them all.

**Reply by kparal** [reply]
OK, thanks. I'll make a PR which points LBA1+2 to the community version of the published repos.
